site/content/api/Tenant-API.md

```diff
@@ -0,0 +1,264 @@
+
+# Tenant Format
+
+Most of the operations of the Tenant API allow or require the inclusion of tenant data in the payload of the
+request or response messages of the operation. Such payload is carried in the body of the corresponding AMQP 
+messages as part of a single *AMQP Value* section.
+
+The tenant data is carried in the payload as a UTF-8 encoded string representation of a single JSON object. It is an error to include payload that is not of this type.
+
+The table below provides an overview of the standard members defined for the JSON object:
+
+| Name                     | Mandatory | Type       | Default Value | Description |
+| :------------------------| :-------: | :--------- | :------------ | :---------- |
+| *tenant-id*              | *yes*     | *string*   |               | The ID of the tenant. |
+| *enabled*                | *no*      | *boolean*  | *true*        | If set to *false* the tenant is currently disabled.|
+| *adapterConfiguration*   | *yes*     | *array*    |               | A list of configuration options valid for certain adapters only. See [Adapter Configuration Format]({{< relref "#adapter-configuration-format" >}}) for details. **NB** This array must contain at least one element - an empty array is handled as an error. **NB** Only one entry with a matching adapter-id is tolerated. If multiple configurations for the same adapter-id are present it is handled as an error.|
+
+Additionally to the specified properties the JSON object MAY contain an arbitrary number of members with arbitrary names which can be of a scalar or a complex type. 
+This allows *clients* to add further information which might be e.g. relevant just for certain adapters only.
+
+### Adapter Configuration Format
+
+When providing / updating a tenant for each *Hono protocol adapter* a configuration must be provided.
+Protocol Adapters without any configuration will be assumed as disabled for that tenant.
+
+
+The table below contains the properties which are used to configure a *Hono protocol adapter*:
+
+| Name                               | Mandatory | Type       | Default Value | Description |
+| :--------------------------------- | :-------: | :--------- | :------------ | :---------- |
+| *adapter-id*                       | *yes*     | *string*   | `null`        | The identification of the adapter which this configuration belongs to |
+| *enabled*                          | *no*      | *string*   | *true*        | If set to false the tenant is not allowed to receive / send data utilizing the given adapter. |
+| *deviceAuthentificationEnabled*   | *no*      | *string*   | *true*        | If set to false devices utilizing the given adapter are not required to authenticate themselves before sending / receiving data through the adapter. |
```

Comment:
this is not about *enabling* authentication but about *requiring* authentication, thus it should be rather `device-authentication-required`

Reply:
adapted